llpc/lower/llpcSpirvLowerResourceCollect.cpp

```diff
@@ -36,7 +36,7 @@
 #include <algorithm>
 // TODO: Fix the code in this file so it does not break the builder abstraction. It should
 // not be including files directly in the LGC directory tree like this.
-#include "../../lgc/builder/BuilderRecorder.h"
+#include "../../lgc/include/lgc/builder/BuilderRecorder.h"
```

Comment:
Can't we replace this with something that avoids the leading `../`?

Reply:
As far as I understand, not without modifying the code of the pass, which is not in the scope of this patch.